tests/test_cumulative_imputation_links.py

```diff
@@ -36,29 +37,38 @@ def test_get_cumulative_links_forward(self, cumulative_links_test_data):
             1,
         )
 
+        actual_output = actual_output.drop(
+            columns=["imputation_group", "missing_value"]
+        )
+
         assert_frame_equal(actual_output, expected_output)
 
     def test_get_cumulative_links_backward(self, cumulative_links_test_data):
         input_data = cumulative_links_test_data.drop(
-            columns=["cumulative_backward_imputation_link", "imputation_group"]
-        )
-
-        expected_output = cumulative_links_test_data[
-            [
-                "imputation_group",
+            columns=[
                 "cumulative_backward_imputation_link",
+                "cumulative_forward_imputation_link",
+                "imputation_group",
             ]
-        ]
+        )
+
+        expected_output = cumulative_links_test_data.drop(
+            columns=["imputation_group", "cumulative_backward_imputation_link"]
+        )
 
         actual_output = get_cumulative_links(
             input_data,
-            "b",
+            "f",
             "strata",
             "reference",
             "target",
             "period",
-            "backward_imputation_link",
+            "forward_imputation_link",
```

Comment:
It there a reason this has changed from backward to forward?

Reply:
Good spot, this should be backward, will update it